packages/core/src/components/MaterialTable/MaterialTable.stories.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import MaterialTable from './MaterialTable';
+import InfoCard from '../../layout/InfoCard';
+
+export default {
+  title: 'Material Table',
+  component: MaterialTable,
+  decorators: [story => <InfoCard title="Table">{story()}</InfoCard>],
+};
+
+const generateTestData: (number) => Array<{}> = (rows = 20) => {
+  const data: Array<{}> = [];
+  while (data.length <= rows) {
+    data.push({
+      col1: `Some value ${data.length}`,
+      col2: `More data ${data.length}`,
+      number: Math.floor(Math.random() * 1000),
+      date: new Date(Math.random() * 10000000000000),
+    });
+  }
+
+  return data;
+};
+
+const testColumns = [
+  {
+    title: 'Column 1',
+    field: 'col1',
+    headerStyle: {
```

Comment:
This is ideally not something we should expose (I think -- @katz95?).

Reply:
+1, at least not encourage it imo

Reply:
+1. It's mostly a easy way to show how to do this in material-table. I guess we could create a layer between the headers sent in the our component and what's passed down to material-table. Might be some other way, but i couldn't find it.

Reply:
Should at least not show it in the examples :)